demo/app/main-view-model-android.ts

```diff
@@ -0,0 +1,42 @@
+import {Observable} from "data/observable";
+var loadingIndicator = require("nativescript-loading-indicator");
+
+export class LoadingAndroidTest extends Observable {
+  
+  public showLoader() {
+    loadingIndicator.show({ message: 'Loading test...' });
+    this.demoLoader();
+  }
+  
+  public showLoaderIndeterminate() {
+    loadingIndicator.show({ indeterminate: true });
+    this.demoProgress();
+  }
+  
+  private demoLoader() {
+    setTimeout(() => {
+      loadingIndicator.hide();
+    }, 3000);
+  }
+  
+  private demoProgress() {
+    setTimeout(() => {
+      loadingIndicator.show({progress: 15});
```

Comment:
Could we space these out like the above so it would read `loadingIndicator.show({ progress: 15 });`